alvr/client_core/src/platform/mod.rs

```diff
@@ -97,7 +103,10 @@ pub fn platform() -> Platform {
             ("Oculus", _, "seacliff") => Platform::QuestPro,
             ("Oculus", _, _) => Platform::QuestUnknown,
             ("Pico", "Pico Neo 3" | "Pico Neo3 Link", _) => Platform::PicoNeo3,
-            ("Pico", _, _) => Platform::Pico4,
+            ("Pico", _, "phoenix") => Platform::Pico4,
```

Comment:
Where did you get those codenames from?

Reply:
A sketchy french driver website lol

Reply:
I looked into the download filename https://www.touslesdrivers.com/index.php?v_page=12&v_code=2301

Reply:
it's correct for Pico 4 Ultra so I think it's correct also for Pico 4 and G3

Reply:
Pico 4 is in fact phoenix, I imagine the other two are correct too